hw/dv/data/xcelium/xcelium.hjson

```diff
@@ -70,12 +92,23 @@
       name: xcelium_waves
       is_sim_mode: 1
     }
-    // TODO support coverage for xcelium
     {
       name: xcelium_cov
       is_sim_mode: 1
-      build_opts: []
-      run_opts:   []
+      build_opts: [// Enable the required cov metrics.
+                   "-coverage {cov_metrics}",
+                   // Limit the scope of coverage collection to the DUT.
+                   "-covdut {dut}",
+                   // Set the coverage configuration file.
+                   "{xcelium_cov_cfg_file}"]
+      run_opts:   [// Coverage database output location.
+                   "-covworkdir {cov_work_dir}",
+                   // Set the scope to the build mode name.
+                   "-covscope {build_mode}",
+                   // Test coverage dir name to create under cov_db_dir.
+                   "-covtest {cov_db_test_dir_name}",
+                   // Overwrite test coverage data if dir already exists.
+                   "-covoverwrite"]
```

Comment:
@rasmus-madsen, please confirm if these look ok.

Reply:
@sriyerg 
sorry if I was not specific enough.
yep they are fine.
you are running with all metrics on

you create the test name.seed a bit differently than I suggested.
xcelium can auto append the seed. but I think this should work also

Reply:
I think that is resonable :D